nesta/core/orms/arxiv-eu_es_config.json

```diff
@@ -53,7 +53,7 @@
                     },
                     "type": "text"
                 },
-                "terms_nuts0_article": {
+                "terms_fieldsOfStudy_article": {
```

Comment:
Note: unrelated changes

Reply:
I ran the `sort_all_json` script, sorry. at least all the jsons are sorted :)